charts/values.yaml

```diff
@@ -0,0 +1,93 @@
+# Default values for marklogic server.
+
+# Number of Marklogic nodes
+replicaCount: 1
+
+# Marklogic image parameters
+image:
+  repository: store/marklogicdb/marklogic-server
+  tag: 10.0-8.1-centos-1.0.0-ea2
```

Comment:
Before pushing lets make sure to update to the latest version, either ea3 or ea4 depending on when it gets merged.

Reply:
updated with ea3 for now